models/rubicon/optimism/rubicon_optimism_schema.yml

```diff
@@ -197,7 +193,7 @@ models:
       - &bought_amount_raw
         name: bought_amount_raw
         description: "the raw amount of the token that the offer has bought"
-      - &sell_amount_usd
+      - &sell_amount_usd: 
```

Comment:
accident to add `:` here?
i can't recall exact scenario, but this caused issues in the past and we had to remove it